MalApi/Manga/MalMangaType.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace MalApi
+{
+    public enum MalMangaType
+    {
+        Unknown = 0,
+        Manga = 1,
+        Novel = 2,
+        OneShot = 3,
+        Doujin = 4, // not sure
```

Comment:
Why aren't you sure about Doujin and OEL?

Reply:
Because I couldn't find such entries when I added that. I found a doujin and confirmed that it returns 4, leaving OEL wtih 7, so this is confirmed OK now.

Reply:
I didn't bother finding a MAL entry to verify it at the time of writing. I went back and found a doujin and confirmed that it returns 4. That leaves OEL as for 7, so this is OK.